src/main/java/org/lantern/Launcher.java

```diff
@@ -279,26 +279,11 @@ void launch() {
             }
         }
         
-        proxyTracker = instance(ProxyTracker.class);
+        //proxyTracker = instance(ProxyTracker.class);
         httpClientFactory = instance(HttpClientFactory.class);
 
         s3ConfigFetcher = instance(S3ConfigFetcher.class);
         
-        if (checkFallbacks) {
```

Comment:
And unless I'm searching incorrectly, with this block of code removed, FallbackChecker is no longer being used anywhere now, which is why I assumed we were removing fallback server testing entirely with this update.